strictdoc/backend/sdoc/writer.py

```diff
@@ -130,8 +131,39 @@ def write(self, document: Document):
                     output += requirement_in_toc
                     output += "\n"
 
+                if default_view is not None:
+                    output += "  DEFAULT_VIEW: "
+                    output += default_view
+                    output += "\n"
+
+        document_view = document.view
+        if document_view:
+            views = document_view.views
+            assert len(views) > 0
+            output += "VIEWS:"
+            output += "\n"
+            for view in views:
+                output += f"- ID: {view.view_id}\n"
+                if view.name is not None:
+                    output += f"  NAME: {view.name}\n"
+                assert len(view.tags) > 0
+                output += "  TAGS:\n"
+                for tag in view.tags:
+                    output += f"  - OBJECT_TYPE: {tag.object_type}\n"
+                    output += "    VISIBLE_FIELDS:\n"
+                    for field in tag.visible_fields:
+                        output += f"    - NAME: {field.name}\n"
+                        placement = field.placement
+                        if placement:
+                            output += f"      PLACEMENT: {placement}\n"
+                hidden_tags = view.hidden_tags
+                if len(hidden_tags) > 0:
+                    output += "  HIDDEN_TAGS:\n"
+                    for hidden_tag in hidden_tags:
+                        output += f"  - {hidden_tag.hidden_tag}\n"
+
         assert document.grammar is not None
-        document_grammar: DocumentGrammar = document.grammar
```

Comment:
This change looks strange. Was there a reason to do it?

Reply:
None at all, for some reason I thought this was my change.